language/move-analyzer/src/symbols.rs

```diff
@@ -191,6 +212,125 @@ pub struct SymbolicatorRunner {
     mtx_cvar: Arc<(Mutex<RunnerState>, Condvar)>,
 }
 
+impl fmt::Display for IdentType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> std::fmt::Result {
+        match self {
+            Self::RegularType(t) => {
+                // Technically, we could use error_format function here to display the "regular"
+                // type, but the original intent of this function is subtly different that we need
+                // (i.e., to be used by compiler error messages) which, for example, results in
+                // verbosity that is not needed here.
+                //
+                // It also seems like a reasonable idea to be able to tune user experience in the
+                // IDE independently on how compiler error messages are generated.
+                write!(f, "{}", type_to_ide_string(t))
+            }
+            Self::FunctionType(mod_ident, name, type_args, args, ret, acquires) => {
+                let type_args_str = if !type_args.is_empty() {
+                    let mut s = "<".to_string();
+                    s.push_str(type_list_to_ide_string(type_args).as_str());
```

Comment:
You can also do `s.push_str(&type_list_to_ide_string(type_args))` or `s += &type_list_to_ide_string(type_args)` if you want. But whatever floats ya boat :)